parser/src/genquery2_sql.cpp

```diff
@@ -1085,19 +1133,33 @@ namespace irods::experimental::api::genquery
 	auto to_sql(const select& _select, const options& _opts) -> std::tuple<std::string, std::vector<std::string>>
 	{
 		try {
+#ifndef IRODS_ENABLE_42X_COMPATIBILITY
 			log_gq::set_level(irods::experimental::log::get_level_from_config("genquery2"));
+#endif // !defined(IRODS_ENABLE_42X_COMPATIBILITY)
 
 			gq_state state;
 
+#ifdef IRODS_ENABLE_42X_COMPATIBILITY
+			rodsLog(LOG_DEBUG, "### PHASE 1: Gather");
```

Comment:
Should this be a higher level (e.g. `LOG_DEBUG8`, `LOG_DEBUG9`) to match the `trace`?

Reply:
It could. Although, that means the genquery2 log messages are more difficult to see due to there being more log messages in general.

Do you feel these messages should require a later debug level?

Reply:
I don't feel strongly about it. It's just that there is a difference in log level between the 4.3.x and 4.2.x versions, and I wanted to make sure that was intentional. Seeing as it was, it seems fine to me. Feel free to resolve

Reply:
This was intentional.

But the more I look at this, it does feel like it should be bumped.

What level have you been using for trace messages?

Reply:
Decided to use LOG_DEBUG for debug values and LOG_DEBUG8 for trace messages.